### PR TITLE
gtksourceview5 5.0.0 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gtksourceview5.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class Gtksourceview5 < Formula
```
Comment (SMillerDev) [inline Formula/gtksourceview5.rb:1]: Does this name follow our standards? https://docs.brew.sh/Versions#acceptable-versioned-formulae

Comment (tschoonj) [reply Formula/gtksourceview5.rb:1]: well there are already `gtksourceview`, `gtksourceview3` and `gtksourceview4` formulas so it made sense to me name it `gtksourceview5`.

Comment (carlocab) [reply Formula/gtksourceview5.rb:1]: Actually, it looks like this formula should be the one called `gtksourceview4` (as it's the one that depends on `gtk4`). At least that's the only way I can make sense of the naming convention for `gtksourceview` and `gtksourceview3`.

We can call it `gtksourceview4@5`? (Just kidding; let's not.)

**Comment by tschoonj** [comment]
Apologies, I forgot about this PR. If the current name is not acceptable, what would be?

**Comment by carlocab** [comment]
What do other distros call it? We should probably also settle on a proper naming convention for the various `gtksourceview` formulae.

**Comment by tschoonj** [comment]
Very few distros are packaging this one yet, but it would [appear](https://repology.org/project/gtksourceview/versions) that usually my suggested name, as well as the one used for the older major releases is followed.

**Review by carlocab — DISMISSED** [review]
I guess `gtksourceview5` is fine. It doesn't conform to Homebrew naming conventions (which I would guess require that this one be called `gtksourceview` and the others `gtksourceview@x`), but I think that ship sailed a while ago. This is the name that makes sense given the formulae we have now.

**Comment by tschoonj** [comment]
@carlocab would you mind merging this one in?

Thanks!

**Comment by tschoonj** [comment]
should I rebase first?

**Comment by carlocab** [comment]
Yea, I was thinking that would help. About to do that.

**Review by carlocab — APPROVED** [review]

**Comment by tschoonj** [comment]
@carlocab try again?

**Comment by tschoonj** [comment]
Thanks!!

**Pull request closed without merging**